src/cli/backup.rs

```diff
@@ -10,6 +10,11 @@ use std::io::copy;
 const TYPE: &str = "application/octet-stream";
 
 pub fn init(matches: &clap::ArgMatches) -> Result<(), Error> {
+	// Initialize o11y
+	crate::o11y::builder().with_log_level(
+		matches.get_one::<String>("log").unwrap()
+	).init();
+
```

Comment:
The `--log` argument isn't customisable on the `surreal backup` command. It should probably be set to `error` though!

Reply:
This was probably a bug! We probably want `error` here too, just like `import` and `export`!

Reply:
yeah, I think this piece of code confused me and I assumed the `--log` arg was global. Will fix it!

Reply:
will fix!